<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogAjoutCheque</class>
 <widget class="QDialog" name="DialogAjoutCheque">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>351</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DialogAjoutCheque</string>
  </property>
  <widget class="QLabel" name="compte">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>76</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Compte</string>
   </property>
  </widget>
  <widget class="QLabel" name="taux">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>76</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Taux</string>
   </property>
  </widget>
  <widget class="QLabel" name="solde">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>76</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Solde</string>
   </property>
  </widget>
  <widget class="QLabel" name="description">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Description</string>
   </property>
  </widget>
  <widget class="QLabel" name="nbTransac">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Nb Transaction</string>
   </property>
  </widget>
  <widget class="QLabel" name="tauxMin">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>76</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Taux min</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="noCompteCheque">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string> 9000</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tauxInteretCheque">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>70</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>0.2</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="soldeCheque">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>120</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>300</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="descriptionCheque">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>170</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string> Cheque</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nbTransactions">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>210</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>20</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tauxMinimum">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>260</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>0.1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="OK">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>310</y>
     <width>112</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>OK</sender>
   <signal>clicked()</signal>
   <receiver>DialogAjoutCheque</receiver>
   <slot>slotAjoutCheque()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>235</x>
     <y>325</y>
    </hint>
    <hint type="destinationlabel">
     <x>250</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotAjoutCheque()</slot>
 </slots>
</ui>
